python/sdk/prelude_sdk/controllers/iam_controller.py

```diff
@@ -108,11 +108,11 @@ def delete_user(self, handle):
             return res.json()
         raise Exception(res.text)
 
-    def reset_password(self, handle: str, account_id: str = None):
+    def reset_password(self, email: str, account_id: str = None):
```

Comment:
Make sure you change the call in the CLI too

Reply:
its actually already email,
<img width="902" alt="Screenshot 2023-08-29 at 6 58 29 PM" src="https://github.com/preludeorg/libraries/assets/84744117/089d61bc-5873-4b6d-9d13-c3fafd1ef9a8">

Reply:
Handle=email

Change to

Email=email

Reply:
oh shoot lol

Reply:
fixed